library/cartridge_edit_topology.py

```diff
@@ -44,12 +51,48 @@
 ''' % (helpers.FORMAT_REPLICASET_FUNC, helpers.FORMAT_SERVER_FUNC)
 
 
-def get_configured_replicasets(module_hostvars, play_hosts):
-    replicasets = {}
+###############################################
+# Collect information about inventory cluster #
+###############################################
+
+
+def get_all_instances_from_inventory(module_hostvars):
+    instances = {}
+
+    for instance_name, instance_vars in module_hostvars.items():
+        if helpers.is_stateboard(instance_vars):
+            continue
+
+        instance = {}
 
-    for instance_name in play_hosts:
-        instance_vars = module_hostvars[instance_name]
+        if helpers.is_expelled(instance_vars):
+            instance['expelled'] = True
+        else:
+            if instance_vars.get('zone') is not None:
+                instance['zone'] = instance_vars['zone']
+            if instance_vars.get('config') is not None:
+                if 'advertise_uri' in instance_vars['config']:
+                    instance['uri'] = instance_vars['config']['advertise_uri']
+
+        if instance:
+            instances[instance_name] = instance
 
+    return instances
+
+
+def get_instances_to_configure(module_hostvars, play_hosts):
+    module_hostvars = {
+        instance_name: instance_vars
+        for instance_name, instance_vars in module_hostvars.items()
+        if instance_name in play_hosts
+    }
+    return get_all_instances_from_inventory(module_hostvars)
```

Comment:
`get_all_instances_from_inventory` -> `get_all_instances_from_hostvars`? Generally nventory is all hostvars.

Reply:
No, inventory is a common term, hostvars is the name of a variable